### DISCUSSION TITLE
Race change - keep racials (PVP)

### DISCUSSION THREAD

**Discussion by Licse** [chromiecraft/chromiecraft#5279] [Suggestions]
CC is cross faction server, and it solves the racial PVE problem. 
tho for the PVP arena aspect it still needs adjustments, i suggest one of the following:
1. Enable (for CC points) visual race changer but keep racials as before. - obv u can choose only the races available to the class.
example: i am a human mage, i hate its looking, i choose undead xmog. my human racials stay- visually undead.
3. Disable all racials in arena as they give a significant advantage and force players to choose one just for performance.(bad game design)

as for point 1, Bench already made a script with npc that u can apply if chosen.

**Comment by balleny** [comment]
if you consider 2, you would also need to modify the virtual hitboxes of male Taurens, Draenei and gnomes, to be the same sized hitbox and hitrange as all other races.

**Comment by Licse** [comment]
im pretty sure hit box was different only in classic and tbc, they changed it(all races have same hit boxes) in late tbc or wotlk due to taurans hitting thru pillars in arena.

**Comment by cachocarne** [comment]
Probably it would be much more simple to find an addon for your client. I played on servers with race swaps and it was kind of annoying, resulting in other problems such as having to announce on text box on every arena what is the "real" race of every adversary. And of course, it was not on BGs or anywhere else. A client-side visual modificator should be enough. If it exists or not, I don't know.

**Comment by heyitsbench** [comment]
> as for point 1, Bench already made a script with npc that u can apply if chosen.

No I didn't.

**Comment by Never6** [comment]
Racials aren't that impactful. 
If you can't get r1 as orc you won't get as human
Racials in 2.4.3 are way more impactful

**Comment by xusbass11-hub** [comment]
I'm VERY much in favor of this proposal. Why do I have to become human (double trinket) to be competitive in PvP? Why do I have to make my warrior an orc to be competitive in PvE if I love the undead? Why do I have to make my mage a troll to be competitive in PvE if I love the undead? Everyone should be able to play with the racial appearances they like best. Thank you.

**Comment by schanzee** [comment]
I would also prefer to free pick racials so you don't have to choose between minmaxing and aesthethics (which plays an even bigger role here with transmogrification. Best racials even vary depending on spec...

Make the visuals match with the chosen racials in arena if that's a concern.
